src/psyclone/tests/psyir/nodes/node_test.py

```diff
@@ -548,37 +548,8 @@ def test_node_dag(tmpdir, have_graphviz):
     assert "unsupported graphviz file format" in str(excinfo.value)
 
 
-def test_find_symbol_table():
```

Comment:
Did you mean to remove this test? There's still a `Node.find_symbol_table()` method.

Reply:
No, that was a mistake by me when merging together code that clashed. Thanks for noticing. I've added it back in now.